com.sudovanilla.penpot-desktop.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: com.sudovanilla.penpot-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+
+modules:
+  - name: penpot-desktop
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://sudovanilla.com/distribute/applications/penpot-desktop/latest/Penpot-Desktop.AppImage
+        sha256: d448a70f45cef03a89efe2f5c28b1dae2a0a53cf627749865195bec422e7493a
+      - type: file
+        path: ./run.sh
+      - type: file
+        path: ./penpot-desktop.desktop
+      - type: file
+        path: com.sudovanilla.penpot-deskop.appdata.xml
+    build-commands:
+      - install -Dm755 ./run.sh /app/bin/run.sh
+      - mv icon128.png /app/share/icons/hicolor/128x128/apps/io.vikunja.Vikunja.png
```

Comment:
this is the wrong name for the icon.
(same below)

ALSO isn't there an icon in the appimage to use instead?

Reply:
There is `squashfs-root/penpot-desktop.png`

Please remove the icons from this submission and use the one extracted from the app image installed in the proper location with the proper name.

Reply:
Sorry about that, the AppImage file provided has been extracted on my machine and the icon and desktop entry from it is now being used.